src/js/signin/containers/Signin.jsx

```diff
@@ -0,0 +1,143 @@
+                  <h5>Sign in with an existing account</h5>
+                  <div>
+                    <button className="usa-button-primary" onClick={this.handleLogin}>
+                      <img alt="ID.me" src="/img/signin/mhv-icon.svg"/><strong> Sign in with My HealtheVet</strong>
+                    </button>
+                    <button className="dslogon" onClick={this.handleLogin}>
+                      <img alt="ID.me" src="/img/signin/dslogon-icon.svg"/><strong> Sign in with DS Logon</strong>
+                    </button>
+                    <button className="usa-button-primary va-button-primary" onClick={this.handleLogin}>
+                      <img alt="ID.me" src="/img/signin/idme-icon-white.svg"/><strong> Sign in with ID.me</strong>
+                    </button>
+                    <span className="sidelines">OR</span>
+                    <div className="alternate-signin">
+                      <h5>Don't have those accounts?</h5>
+                      <button className="idme-create usa-button usa-button-outline" onClick={this.handleSignup}>
+                        <img alt="ID.me" src="/img/signin/idme-icon-dark.svg"/><strong> Create an ID.me account</strong>
+                      </button>
+                      <p>Use your email, Google, or Facebook</p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className="columns usa-width-one-half medium-6">
+              <div className="explanation-content">
+                <h2 className="hide-for-small">Manage the benefits and services you've earned. Simply and securely.</h2>
+                <p>
+                  With Vets.gov you can:
+                </p>
+                <ul>
+                  <li>Track your VA claims and appeals</li>
+                  <li>Download your VA health record</li>
+                  <li>Refill your VA prescriptions</li>
+                  <li>And more</li>
+                </ul>
+                <p>
+                  <strong>A secure account powered by ID.me</strong><br/>
+                  ID.me is our trusted technology partner in helping to keep your personal information safe. They specialize in digital identity protection and help us make sure you're you—and not someone pretending to be you—before we give you access to your information.
+                </p>
+                <p><a href="#">Learn more about ID.me</a></p>
+              </div>
+            </div>
+          </div>
+          <div className="row">
+            <div className="columns small-12">
+              <div className="help-info">
+                <h4>Having trouble signing in?</h4>
+                <p><a href="/faq">Get answers to Frequently Asked Questions</a></p>
+                <p>
+                  Call the Vets.gov Help Desk at <a href="tel:+18555747286">1-855-574-7286</a> (TTY: <a href="tel:+18008294833">1-800-829-4833</a>).<br/>
```

Comment:
Hm... Super small thing, but we have inconsistencies about having spaces between the days and times and the hyphens. Should make sure they're all the same at some point. @melwoodard

Reply:
I think this is the format that the content team prefers. We can circle back at a later point to address any that are inconsistent with this